tests/tensor/test_basic.py

```diff
@@ -848,10 +848,29 @@ def test_zeros(self):
             inp = np.zeros(shp, dtype=config.floatX)
             assert np.allclose(zeros_tensor(inp), np.zeros(shp))
 
-    def test_full(self):
-        full_pt = ptb.full((2, 3), 3, dtype="int64")
+    @pytest.mark.parametrize("shape", [(2, 3), 5, np.int32(5), np.array(5)])
+    def test_full(self, shape):
+        full_pt = ptb.full(shape, 3, dtype="int64")
         res = pytensor.function([], full_pt, mode=self.mode)()
-        assert np.array_equal(res, np.full((2, 3), 3, dtype="int64"))
+        assert np.array_equal(res, np.full(shape, 3, dtype="int64"))
+
+    def test_full_with_scalar(self):
```

Comment:
You can use the previous test machinery with `pt.constant(5)`

Reply:
Thanks! was missing that